src/main/java/core/basesyntax/service/FileWriter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FileWriter {
+    void write(String report);
```

Comment:
Consider adding a parameter to the `write` method for specifying the file path. This will make the interface more flexible and reusable for writing to different files.